### PR TITLE
Values3 gt pr14

### PR DESCRIPTION
**Pull request by TalbotG**
Trying again to submit 
calc-numbers.html 
calc-percent-number-line-height-crash.html 
reference/calc-percent-number-line-height-crash-ref.html

### CODE REVIEW

**Comment by TalbotG** [comment]
If pull request 15192 succeeds, then I will close pull request 15103 immediately.

File: css/css-values/calc-numbers.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+ /* verifyComputedStyle(property_name, initial_value, specified_value, expected_value, description) */
+
+    verifyComputedStyle("tab-size", "initial", "calc(2 * 3)", "6", "testing tab-size: calc(2 * 3)");
+
+    verifyComputedStyle("tab-size", "12345", "calc(2 * -4)", "12345", "testing tab-size: calc(2 * -4)");
+
+    verifyComputedStyle("opacity", "initial", "calc(2 / 4)", "0.5", "testing opacity: calc(2 / 4)");
+
+    verifyComputedStyle("tab-size", "12345", "calc(2 / 4)", "12345", "testing tab-size: calc(2 / 4)");
```
Comment (frivoal) [inline css/css-values/calc-numbers.html]: tab-size accepts numbers, not integers.
```suggestion
    verifyComputedStyle("tab-size", "12345", "calc(2 / 4)", "0.5", "testing tab-size: calc(2 / 4)");
```

**Review by frivoal — CHANGES_REQUESTED** [review]
One change requested.

As for the crash test, it's not wrong, but I am not sure it is useful.

**Review by frivoal — APPROVED** [review]
LGTM

**Comment by TalbotG** [comment]
See Pull Request 15226 for such correction

Credit should go to Emilio Cobos Álvarez ( @emilio ) for his feedback on this

**Pull request merged**